packages/cli/src/workflows/workflows.services.ts

```diff
@@ -152,14 +152,7 @@ export class WorkflowsService {
 			return [];
 		}
 
-		const fields: Array<keyof WorkflowEntity> = [
-			'id',
-			'name',
-			'active',
-			'createdAt',
-			'updatedAt',
-			'nodes',
```

Comment:
`nodes` are actually needed [here](https://github.com/n8n-io/n8n/blob/master/packages/cli/src/workflows/workflows.services.ee.ts#L118).
We should definitely only be querying minimum data needed for `GET /rest/workflows`, but that required changing the workflow table to have additional columns to catch the data that's currently being augmented in `addCredentialsToWorkflows`
@krynble Can you add more info here please?

Reply:
I see, I'll just remove it from the response because the frontend doesn't need `nodes` in the list

Reply:
We want to actually get rid of the `sharedWith` as part of the list. 

It is necessary when fetching a single workflow, but when returning multiple workflows it's not necessary.

Also it's worth checking why we're returning the credentials part to FE. If the FE does not need it, we can simplify this call too.

Reply:
As far as I have seen the list needs only the `owner` to be defined (it wouldn't even need owner props but we can't just strip them out because of TS interface)

We could also remove `EEWorkflows.addCredentialsToWorkflows` call [here](https://github.com/n8n-io/n8n/blob/master/packages/cli/src/workflows/workflows.controller.ee.ts#L207)